### ISSUE TITLE
Management page not responsive (table)

### ISSUE THREAD

**Issue by marcwieland95** [stcr/subscribe-to-comments-reloaded#433]
This is right? The table in the management is not responsive at all?

If this is really the case I could prepare a pull request to fix this issue.

**Comment by jnorell** [comment]
Indeed, a quick test shows that is the case, it gets pretty squished together on the screen.  I'd say prepare that pull request!  :)

**Comment by marcwieland95** [comment]
Meanwhile, I created my own plugin because this one can too much for our use case. Maybe I find the old code I played with and can create a pull request.





  | meanwhile
-- | --
